### PR TITLE
Add TestHelloWorld to set up prow, and add more loggings for boskos project rental.

### PR DESCRIPTION
**Pull request by sawsa307**
Add TestHelloWorld to set up prow, and add more loggings for boskos project rental.

### CODE REVIEW

File: test/utils.go [hunk 1]
```diff
@@ -43,12 +43,13 @@ func getBoskosProject(resourceType string) *common.Resource {
...
 		wait.Backoff{
-			Duration: time.Second,
+			Duration: time.Minute,
 			Factor:   1.0,
-			Steps:    10,
+			Steps:    30,
 		},
 		func(err error) bool { return true },
 		func() error {
+			klog.Info("Trying to acuquire boskos project...")
```
Comment (bowei) [inline test/utils.go:52]: spelling  "acuquire"

File: test/utils.go [hunk 2]
```diff
@@ -60,7 +61,7 @@ func getBoskosProject(resourceType string) *common.Resource {
 		},
 	)
 	if err != nil {
-		klog.Fatalf("timed out trying to acquire boskos project")
+		klog.Fatalf("timed out trying to acquire boskos project: %v", err)
```
Comment (bowei) [inline test/utils.go:64]: Error trying to acquire boskos: %v

File: test/utils.go [hunk 3]
```diff
@@ -43,12 +43,13 @@ func getBoskosProject(resourceType string) *common.Resource {
 	var project *common.Resource
 	err := retry.OnError(
 		wait.Backoff{
-			Duration: time.Second,
+			Duration: time.Minute,
 			Factor:   1.0,
-			Steps:    10,
+			Steps:    30,
```
Comment (bowei) [inline test/utils.go:48]: This is 30 minutes. Do we want to wait this long? also should the retry by faster?

Why not do:

10*time.Second

Steps 30

Do we have evidence that the values you are recommending make sense

File: test/utils.go (hunk 1 above)
Comment (sawsa307) [reply test/utils.go:52]: Updated.

File: test/utils.go (hunk 2 above)
Comment (sawsa307) [reply test/utils.go:64]: Updated.

File: test/utils.go (hunk 3 above)
Comment (sawsa307) [reply test/utils.go:48]: I don't see anywhere suggesting an estimate of how long it takes, but this is the setup using in https://github.com/GoogleCloudPlatform/gcs-fuse-csi-driver/blob/7485ad00b362a6172c2bceddcdfba8688a8be0e6/test/e2e/utils/project.go#L35. I think we can start with 5 mins and increase if that's not enough.

**Review by bowei — APPROVED** [review]

**Pull request merged**